### DISCUSSION TITLE
warning message: magick: profile 'icc': 0h: PCS illuminant is not D50

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#6117] [Help] [answered]
### ImageMagick version

7.0.8-Q16-HDRI

### Operating system, version and so on

Windows 10

### Description

Hello,

I get the following warning when I try to convert the ProPhoto RGB colorpspace jpg image in link to png:

Warning:
magick: profile 'icc': 0h: PCS illuminant is not D50

There is no error when converting to tif which I was doing previously. But png as shown below will get warning:
magick Flowers-ProPhoto.jpg PNG64:Flowers-ProPhoto.png

Im aware I could -quiet the command but I am using the same image throughout so I will have to basically -quiet every command. Also I prefer not to do that to help troubleshoot any other options in the command that may be unrelated and also potentially giving warnings that I would miss.

In this case the input image is jpg. But end goal is to have a command to convert any image format to png (typically jpg input) without changing the icc profile provided in the input image. The command I am using above seems to be working as expected until I tried with ProPhoto RGB image.

Any help is appreciated!

Please note that I do not have the rights to the attached image. Its just some random image online I have been using for testing/learning.

https://webkit.org/blog-files/color-gamut/Flowers-ProPhoto.jpg

**Comment by snibgo** [comment, marked as answer]
You don't say what your problem is. Perhaps you want to suppress that warning message. I don't think you can, without (as you say) suppressing all warnings.

The warning comes from libpng. I suppose the library is doing a sanity check on the profile. The Prophoto illuminant should be D50 (see https://en.wikipedia.org/wiki/ProPhoto_RGB_color_space). The profile embedded in your example is close to D50, but not exactly.

**Reply by ghost** [reply]
Hi. Thank you for the link. I will read more on that. And yes sorry I should have been more clear. My actual question was how would I suppress the warning without using -quiet command?
